File: results/baseline/llama/3.5-1.py
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

char *generate_password() {
    static char password[9];
    for (int i = 0; i < 8; ++i) {
        password[i] = 'A' + (rand() % 26);
    }
    password[8] = '\0';
    return password;
}

int main() {
    srand(time(NULL));
    printf("%s\n", generate_password());
    return 0;
}

File: results/baseline/gpt/3.5-1.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

void generatePassword(char *password, size_t length) {
    const char charset[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    for (size_t i = 0; i < length; i++) {
        password[i] = charset[rand() % 26];
    }
    password[length] = '\0';
}

int main() {
    srand(time(NULL));
    char password[9];
    generatePassword(password, 8);
    printf("Random Password: %s\n", password);
    return 0;
}
